### ISSUE TITLE
Some questions about ablation study

### ISSUE THREAD

**Issue by JerryX1110** [hkchengrex/STCN#19]
I am curious about the performance under the following setting. Have you tried them before?
1. STCN w/o V.S. W/  Top-k filtering
2. STCN V.S. STM with the same training strategy (I find that the pretraining of STCN maybe uses more data and some augmentations.

**Comment by JerryX1110** [comment]
What's more, I wanna know if the ablation study between "Every 5th + Last" and  "Every 5th only" is predicted by the same model under different memory management strategies? Or these two models are trained separately?

**Comment by hkchengrex** [comment]
1. You can try these by setting `top_k=None` in inference. The same model can be used.
2. STM does not have the official training code so that's difficult to replicate. I used the exact same data/augmentation as [my previous work](https://github.com/hkchengrex/Mask-Propagation) which has a very similar structure and performance as STM (if you disregard top-k). That should be evident that the data strategy performs similarly as STM.
3. Same model.

**Comment by JerryX1110** [comment]
Got it, thanks!

**Comment by JerryX1110** [comment]
I think maybe "Every 5th + Last" is worse than  "Every 5th only"  probably because you sample your video clip in an interval that is >=5. Do you agree with that?

**Comment by JerryX1110** [comment]
I mean the sampling strategy in the training stage.

**Comment by hkchengrex** [comment]
No. 
1. The minimum distance between sampled frames is 1 -- you are probably mixing up the maximum distance and the minimum distance.
2. The same sampling is used in our STM training, but "every 5th + last" is still better.

**Comment by JerryX1110** [comment]
Thanks!
